config/sys.config.src

```diff
@@ -1,9 +1,21 @@
 [
     {hpr, [
+        {packet_reporter, #{
+            access_key_id => <<"${AWS_ACCESS_KEY_ID}">>,
+            secret_access_key => <<"${AWS_SECRET_ACCESS_KEY}">>,
+            region => <<"${AWS_DEFAULT_REGION}">>,
+            bucket => <<"${PACKET_REPORTER_BUCKET_NAME}">>,
+            interval_duration => 300000,
+            write_dir => "/var/data/hpr/tmp/",
+            max_file_size => 50_000_000,
+            upload_window => 900000,
+            retry_sleep_time => 60000,
+            max_upload_retries => 5
+        }},
         {config_worker, #{
             host => "${HPR_CONFIG_WORKER_HOST}",
             port => "${HPR_CONFIG_WORKER_PORT:-8080}",
-            file_backup_path => "/var/data/config_worker.backup"
+            file_backup_path => "/var/data/hpr/config_worker.backup"
```

Comment:
```suggestion
            file_backup_path => "/var/data/config_worker.backup"
```
This needs to be removed or it will mess things up in container